### PR TITLE
fix logging window function

### PR DESCRIPTION
**Pull request by freeznet**
Master Issue: https://github.com/streamnative/function-mesh/issues/474

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Review by tpiperatgod — APPROVED** [review]

**Comment by jiangpengcheng** [comment]
so update the jdk version can fix the issue?

File: .ci/tests/integration/e2e.yaml [hunk 1]
```diff
@@ -9,9 +9,9 @@ setup:
         PULSAR_IMAGE_TAG=2.9.2.23 PULSAR_IMAGE=streamnative/pulsar-all KIND_PUSH=true images/build.sh
         PULSAR_IMAGE_TAG=2.9.2.23 KIND_PUSH=true images/samples/build.sh
 
-    - name: install helm
-      command: |
-        curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+#    - name: install helm
```
Comment (imaffe) [inline .ci/tests/integration/e2e.yaml:12]: Why do we ignore this install helm step ?

Comment (freeznet) [reply .ci/tests/integration/e2e.yaml]: ahh, just for local testing, forget to revert.

**Comment by imaffe** [comment]
LGTM~

**Review by tpiperatgod — APPROVED** [review]

**Pull request merged**